### PR TITLE
DEVOPS-000: Update HelpSU linky

### PR DESCRIPTION
**Pull request by jbickar**
Waiting for confirmation from UIT Comms on the wording

### CODE REVIEW

**Review by sherakama — APPROVED** [review]
GTG

**Comment by sherakama** [comment]
Please don't forget the link in the README.md

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Install this module like [any other Drupal module](https://www.drupal.org/docume
 
 ## Authentication
 
-Before you get started you will need to have authentication credentials. To get authentication credentials, [file a HelpSU request](https://helpsu.stanford.edu/helpsu/3.0/auth/helpsu-form?pcat=CAP_API&dtemplate=CAP-OAuth-Info) to Administrative Applications/CAP Stanford Profiles.
+Before you get started you will need to have authentication credentials. To get authentication credentials, [submit a service request](https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=3ecd311a13b832008a9175c36144b069) to Administrative Applications/CAP Stanford Profiles.
```
Comment (jbickar) [inline README.md]: `thisisfine.gif`

**Comment by jbickar** [comment]
Tina confirmed the link. Waiting on confirmation from Staci about the wording. If we're only going to update this once every 5 years, might as well get it right.

**Comment by jbickar** [comment]
Thus justifying my continuing employment at Stanford University.

**Comment by jbickar** [comment]
CodeClimate is saying, "The $text argument to l() should be enclosed within t() so that it is translatable"

I thought we weren't supposed to double up on `t()`? Should it be this?

```
'#markup' => t("Please enter your authentication information for the CAP API. If you don't have these credentials yet you can !helpsu.", array('!helpsu' => l(t("submit a Help request"), "https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=3ecd311a13b832008a9175c36144b069"))),
```

**Comment by sherakama** [comment]
Yeah, CC is right. The token replacement won't get translated so the text in the `l()` needs to be translated independently.

**Comment by jbickar** [comment]
I probably knew that at some point, but it likely got replaced by some bit of useless knowledge.

**Pull request merged**